specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/webapplicationfirewall.json

```diff
@@ -726,6 +718,14 @@
           "readOnly": true,
           "type": "string"
         },
+        "defaultState": {
```

Comment:
Add new property in the response is considered a breaking change.

Reply:
@lirenhe - The new fields are read-only and the associated API only supports GET requests (no PUT).

Reply:
adding read-only properties in an api version is ok from ARM guidelines